test/object-store/transaction_log_parsing.cpp

```diff
@@ -1783,9 +1783,16 @@ TEMPLATE_TEST_CASE("DeepChangeChecker", "[notifications]", ListOfObjects, ListOf
         return info;
     };
 
-    _impl::DeepChangeChecker::RelatedTables tables;
+    std::vector<_impl::DeepChangeChecker::RelatedTable> related_tables;
+    std::pair<TableKey, ColKey> pair_int(table->get_key(), table->get_column_key("int"));
+    std::pair<TableKey, ColKey> pair_link(table->get_key(), table->get_column_key("link1"));
```

Comment:
the interesting case to test here is:
```
std::pair<TableKey, ColKey> pair_link(table->get_key(), table->get_column_key(test_type.property.name));
```

Reply:
Thanks for pointing this out. Added with 24d0b49 .